eva_cttv_pipeline/clinvar_record.py

```diff
@@ -172,16 +192,16 @@ def trait_pubmed_refs(self):
 
     @property
     def observed_pubmed_refs(self):
-        pubmedrefsList = []
+        pubmed_refs_list = []
         if 'observedIn' in self.data['referenceClinVarAssertion']:
-            for observedInRecord in self.data['referenceClinVarAssertion']['observedIn']:
-                for observedDataRecord in observedInRecord['observedData']:
-                    if 'citation' in observedDataRecord:
-                        for citationRecord in observedDataRecord['citation']:
-                            if ('id' in citationRecord) and citationRecord['id'] is not None:
-                                if citationRecord['id']['source'] == 'PubMed':
-                                    pubmedrefsList.append(int(citationRecord['id']['value']))
-        return pubmedrefsList
+            for observed_in_record in self.data['referenceClinVarAssertion']['observedIn']:
```

Comment:
This `xxx_record` nomenclature seems a bit confusing to me, because they don't represent records. What do you think about removing the suffix?